src/Surfnet/StepupSelfService/SelfServiceBundle/Service/RemoteVetting/Value/AttributeMatch.php

```diff
@@ -31,17 +31,22 @@ class AttributeMatch
      * @var string
      */
     private $name = '';
+    /**
+     * @var string
+     */
+    private $value = '';
 
     /**
      * @param string $name
      * @param bool $valid
      * @param string $remarks
      */
-    public function __construct($name, $valid, $remarks)
+    public function __construct($name, $value, $valid, $remarks)
     {
         $this->name = $name;
         $this->valid = $valid;
         $this->remarks = $remarks;
+        $this->value = $value;
```

Comment:
I strongly advise adding input guarding to this value object, not only for the recently added value, but for the other fields too!

Reply:
Good suggestion, done.